graphene/src/vec3.rs

```diff
@@ -2,35 +2,36 @@
 
 use crate::Vec3;
 use glib::translate::*;
+use std::fmt;
 
 impl Vec3 {
     #[doc(alias = "graphene_vec3_init")]
-    pub fn new(x: f32, y: f32, z: f32) -> Vec3 {
+    pub fn new(x: f32, y: f32, z: f32) -> Self {
         assert_initialized_main_thread!();
         unsafe {
-            let mut vec = Vec3::uninitialized();
+            let mut vec = Self::uninitialized();
             ffi::graphene_vec3_init(vec.to_glib_none_mut().0, x, y, z);
             vec
         }
     }
 
     #[doc(alias = "graphene_vec3_init_from_vec3")]
     #[doc(alias = "new_from_vec3")]
-    pub fn from_vec3(src: &Vec3) -> Vec3 {
+    pub fn from_vec3(src: &Vec3) -> Self {
```

Comment:
This one is a bit strange. That's basically `clone()` :)

Reply:
Other types also have such functions. Not sure how useful it is to keep these :shrug:

Reply:
I dropped them, we can re-add them if someone complains